langchain/src/prompts/chat.ts

```diff
@@ -27,7 +27,7 @@ export abstract class BaseMessagePromptTemplate {
   serialize(): SerializedMessagePromptTemplate {
     return {
       _type: this.constructor.name,
-      ...JSON.parse(JSON.stringify(this)),
+      ...JSON.parse(JSON.stringify(this))
```

Comment:
If possible, please invoke `yarn format` / Prettier to properly format the code.